cmd/juju/caas/remove.go

```diff
@@ -98,7 +101,7 @@ func (c *RemoveCAASCommand) Init(args []string) (err error) {
 
 // Run is defined on the Command interface.
 func (c *RemoveCAASCommand) Run(ctxt *cmd.Context) error {
-	if c.ControllerName == "" && !c.Local {
+	if c.ControllerName == "" && !c.ClientOnly {
 		return errors.Errorf(
 			"There are no controllers running.\nTo remove cloud %q from the current client, use the --client option.", c.cloudName)
```

Comment:
--client-only
(here, maybe elsewhere was well)

Reply:
True \o/ Changed. Thank you for the pick up :D